File: Day3/src/gaussian.cpp
#define _USE_MATH_DEFINES
#include <iostream>
#include <math.h>
using namespace std;

double f(double mu,double sigma2,double x){
double sigma = sqrt(sigma2);
double prob=(1/(sigma*sqrt(2*M_PI)))*exp(-0.5*pow(((x-mu)/sigma),2));
return prob;
}

int main(){
cout<<f(10.0,4.0,8.0)<<endl;
return 0;
}

File: Day3/scripts/gaussian.py
import math

def f(mu,sigma2,x):
	sigma=math.sqrt(sigma2)
	prob = (1/(sigma*math.sqrt(2*math.pi)))*(math.exp(-0.5*(math.pow((x-mu)/sigma,2))))
	return prob

if __name__ == '__main__':
	print(f(10.0,4.0,8.0))
